src/Libraries/CoreNodes/Math.cs

```diff
@@ -253,7 +253,7 @@ public static double Asin(double ratio)
         ///     Finds the inverse tangent, the angle whose tangent is the given ratio.
         /// </summary>
         /// <param name="ratio">The tangent of the angle.</param>
-        /// <returns name="angle">The angle whose tangent is the input ratio.</returns>
+        /// <returns name="double">The angle whose tangent is the input ratio (double)</returns>
```

Comment:
`(double)`

Reply:
this goes for all of the following `(double)`

Reply:
adressed in 
0cb98d5